src/Sulu/Bundle/LocationBundle/Resources/docs/index.md

```diff
@@ -72,10 +40,11 @@ You can choose between at least two different endpoints:
 - OSM: http://nominatim.openstreetmap.org/search: [Example](http://nominatim.openstreetmap.org/search?format=json&q=466+dorchester+road+weymouth)
 - MapQuest: http://open.mapquestapi.com/nominatim/v1/search.php: [Example](http://open.mapquestapi.com/nominatim/v1/search.php?format=json&q=466+dorchester+road+weymouth)
 
-Both of the aboce services are free-to-use, but the mapquest endpoint seems faster.
+Both of the above services are free-to-use, but the mapquest endpoint seems faster.
 
 Configuration:
 
+- **api_key**: Optional, but recommended if making many requests ([Developer Mapquest](https://developer.mapquest.com/user/me/apps)).
 - **endpoint**: The endpoint to use
```

Comment:
i guess we can remove this?

Reply:
you could use another provider (this is the official)